pkg/apis/serving/k8s_validation.go

```diff
@@ -872,6 +883,56 @@ func ValidatePodSecurityContext(ctx context.Context, sc *corev1.PodSecurityConte
 	return errs
 }
 
+// warnDefaultContainerSecurityContext warns about Kubernetes default
+// SecurityContext values which are insecure (i.e. the "restricted" profile
+// forbids these values). Because securityContext values may also be set at
+// the Pod level, the container-level settings need to be considered alongside
+// the Pod-level settings.
```

Comment:
nit: the release note is a little clearer in that the warning will sound when these values are unset. would be helpful to include that detail in the comment here, maybe something like:

```suggestion
// warnDefaultContainerSecurityContext warns about Kubernetes default
// SecurityContext values which are unset and thus insecure (i.e. the "restricted" profile
// forbids these values). Because securityContext values may also be set at
// the Pod level, the container-level settings need to be considered alongside
// the Pod-level settings.
```

Reply:
Done (and rewrapped, just 'cuz)